biom/cli/installation_informer.py

```diff
@@ -0,0 +1,99 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2011-2015, The BIOM Format Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from __future__ import division
+
+import sys
+
+
+def show_install_info():
+    lines = []
+    lines.extend(_get_formatted_system_info())
+    lines.extend(_get_formatted_dependency_version_info())
+    lines.extend(_get_formatted_package_info())
+    lines.append('')
+    return '\n'.join(lines)
+
+
+def _get_formatted_system_info():
+    return _format_info(_get_system_info(), 'System information')
+
+
+def _get_formatted_dependency_version_info():
+    return _format_info(_get_dependency_version_info(), 'Dependency versions')
+
+
+def _get_formatted_package_info():
+    return _format_info(_get_package_info(), 'biom-format package information')
+
+
+def _get_system_info():
+    return (("Platform", sys.platform),
+            ("Python/GCC version", sys.version.replace('\n', ' ')),
```

Comment:
Note that we're just refactoring the pyqi code into click -- I'd like to avoid making changes to the code itself at this point b/c due to time constraints (there is a lot of less-than-ideal code we'll be converting over today)